<launch>
    <arg name="pose_file"  default="/home/yudong/Downloads/zhitenglou/poses/01.txt" />
    <arg name="output_bag_file" default="/home/yudong/Downloads/zhitenglou/ztl_re.bag" />
    <arg name="pc_folder" default="/home/yudong/Downloads/zhitenglou/sequences/01/velodyne/" />
    <arg name="start_num" default="4422" />
    <arg name="to_bag"          default="true" />
    <arg name="publish_delay"   default="1" />
    <arg name="reverse" default="true" />

    <node name="plyHelper" pkg="aloam_velodyne" type="plyHelper" output="screen"> 
        <param name="pose_file"         type="string"   value="$(arg pose_file)" />
        <param name="pc_folder"         type="string"   value="$(arg pc_folder)" />
        <param name="start_num"         type="int"   value="$(arg start_num)" />
        <param name="to_bag"            type="bool"     value="$(arg to_bag)" />
        <param name="output_bag_file"   type="string"   value="$(arg output_bag_file)" /> <!-- replace with your output folder -->
        <param name="publish_delay"     type="int"      value="$(arg publish_delay)" />
        <param name="reverse"           type="bool"     value="$(arg reverse)" />
    </node>
</launch>